both/access_control.js

```diff
@@ -231,101 +229,75 @@ if(Meteor.isServer) {
         'createdAt',
       ];
 
-      if(_.difference(fields, allowedFields).length > 0) {
+      return _onlyAllowedFields(fields, allowedFields);
```

Comment:
Not telling you to change this, but to just explain the reasoning behind the previous structure: I wanted to write code that looks like

```javascript

if(shouldn't reason #1)
   return false;

if(shouldn't reason #2)
   return false;
...

return true;
```

Also, return statements that call functions can be hard to analyze with debuggers, especially if the function call has a side effect (not the case here).